java/java-ranger-regression/WBS_prop1/Main.java

```diff
@@ -0,0 +1,305 @@
+				         (WBS_Node_WBS_BSCU_Command_AntiSkidCommand_Normal_Switch < 3))) {
+				      WBS_Node_WBS_BSCU_Switch2 = 2; 
+				   } else { 
+					   if (((WBS_Node_WBS_BSCU_Command_AntiSkidCommand_Normal_Switch >= 3) && 
+					         (WBS_Node_WBS_BSCU_Command_AntiSkidCommand_Normal_Switch < 4)))  {
+					      WBS_Node_WBS_BSCU_Switch2 = 3; 
+					   } else { 
+					      WBS_Node_WBS_BSCU_Switch2 = 4;
+					   }
+				   }
+			   }
+		   }
+	   }
+		//Debug.printPC("PC @fter(WBS_Node_WBS_BSCU_SystemModeSelCmd_Logical_Operator6) = ");
+
+
+
+		if ((WBS_Node_WBS_Green_Pump_IsolationValve_Switch >= 1))  {
+	      WBS_Node_WBS_SelectorValve_Switch = WBS_Node_WBS_Green_Pump_IsolationValve_Switch;
+	   }  else { 
+	      WBS_Node_WBS_SelectorValve_Switch = 0;
+	   }
+	//	Debug.printPC("PC @fter(WBS_Node_WBS_Green_Pump_IsolationValve_Switch >= 1) = ");
+
+
+
+	   if ((WBS_Node_WBS_BSCU_Switch2 == 0)) {
+	      Nor_Pressure = 0; 
+	   }  else { 
+		   if ((WBS_Node_WBS_BSCU_Switch2 == 1)) {
+		      Nor_Pressure = (WBS_Node_WBS_SelectorValve_Switch / 4);
+		   }  else {
+			   if ((WBS_Node_WBS_BSCU_Switch2 == 2)) {
+			      Nor_Pressure = (WBS_Node_WBS_SelectorValve_Switch / 2);
+			   }  else { 
+				   if ((WBS_Node_WBS_BSCU_Switch2 == 3)) {
+				      Nor_Pressure = ((WBS_Node_WBS_SelectorValve_Switch / 4) * 3);
+				   } else { 
+					   if ((WBS_Node_WBS_BSCU_Switch2 == 4)) { 
+					      Nor_Pressure = WBS_Node_WBS_SelectorValve_Switch;
+					   } else { 
+					      Nor_Pressure = 0;
+					   }
+				   }
+			   }
+		   }
+	   }
+
+	//	Debug.printPC("PC before(WBS_Node_WBS_BSCU_Command_PedalCommand_Switch1 == 0) [region 4] = ");
+
+	   if ((WBS_Node_WBS_BSCU_Command_PedalCommand_Switch1 == 0)) {
+	      Alt_Pressure = 0; 
+	   }  else {
+		   if ((WBS_Node_WBS_BSCU_Command_PedalCommand_Switch1 == 1)) {
+		      Alt_Pressure = (WBS_Node_WBS_AS_MeterValve_Switch / 4); 
+		   }  else { 
+			   if ((WBS_Node_WBS_BSCU_Command_PedalCommand_Switch1 == 2)) {
+			      Alt_Pressure = (WBS_Node_WBS_AS_MeterValve_Switch / 2);
+			   } else { 
+				   if ((WBS_Node_WBS_BSCU_Command_PedalCommand_Switch1 == 3)) {
+				      Alt_Pressure = ((WBS_Node_WBS_AS_MeterValve_Switch / 4) * 3);
+				   } else { 
+					   if ((WBS_Node_WBS_BSCU_Command_PedalCommand_Switch1 == 4)) {
+					      Alt_Pressure = WBS_Node_WBS_AS_MeterValve_Switch; 
+					   } else { 
+					      Alt_Pressure = 0;
+					   }
+				   }
+			   }
+		   }
+	   }
+	//	Debug.printPC("PC @fter(WBS_Node_WBS_BSCU_Command_PedalCommand_Switch1 == 0) [Region 4] = ");
+
+
+	   WBS_Node_WBS_rlt_PRE2 = Nor_Pressure; 
+
+	   WBS_Node_WBS_BSCU_rlt_PRE1 = WBS_Node_WBS_BSCU_Switch2; 
+
+	   WBS_Node_WBS_BSCU_SystemModeSelCmd_rlt_PRE = Sys_Mode;
+//	   WBS_Node_WBS_BSCU_SystemModeSelCmd_rlt_PRE += WBS_Node_WBS_BSCU_SystemModeSelCmd_Logical_Operator6 ? 1 : 0;
+
+		// Assertions added by MWW: these are the truth values for "my" version of the model - may not be true for this code, but should be
```

Comment:
It seems like all these assert should explicitly not be checked.
Might it be worth to remove them in this case to keep the test case clean?

Reply:
Done.